### PR TITLE
Cxtb 251 participant directory filter

### PR DESCRIPTION
**Pull request by rolandosantos1**
Added a check for the filters in the Participants page.

### CODE REVIEW

File: tests/cases/case_care_platform_participants.yaml [hunk 1]
```diff
@@ -80,4 +80,93 @@ TestCreateFollowUpTicket:
...
+      App: "desktop"
+  Actions:
+  #Calling Login case
+    - Type: case
+      Value: CarePartnerLogin
+  #Go to participants
+    - Type: click
+      Strategy: xpath
+      Id : $PAGE.care_platform.navigation_participant$
+  #BGet the value to compare later
```
Comment (miguel-hquez) [inline tests/cases/case_care_platform_participants.yaml:99]: little typo

File: tests/cases/case_care_platform_participants.yaml [hunk 2]
```diff
@@ -80,4 +80,93 @@ TestCreateFollowUpTicket:
...
+        - var: LASTNAME_FILTER_LETTER
+          match: "(.*)"
+  #Click the first darkened letter button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_darkened_letter_button$
+  #Verify it was filtered by the selected letter
+    - Type: get_text
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_senior_in_list_strong$
```
Comment (csanchez-nts) [inline tests/cases/case_care_platform_participants.yaml:113]: What is this "strong" at the end of each list?

File: tests/cases/case_care_platform_participants.yaml (hunk 1 above)
Comment (rolandosantos1) [reply tests/cases/case_care_platform_participants.yaml]: Solved

File: tests/cases/case_care_platform_participants.yaml (hunk 2 above)
Comment (rolandosantos1) [reply tests/cases/case_care_platform_participants.yaml:113]: The etiquette that contains the lastname is a Strong etiquette. Each participant row has a Strong etiquette. You can also check it on the page object, that it is calling the strong on the first row of the page.

**Review by miguel-hquez — APPROVED** [review]

**Pull request merged**